extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -475,7 +475,8 @@ export class AmpA4A extends AMP.BaseElement {
             try {
               // Among other things, the signature might not be proper base64.
               return verifySignature(
-                  response.creative, response.signature, publicKeyInfos);
+                  new Uint8Array(response.creative),
```

Comment:
There are *supposed* to be.  They *should* have been failing before.  It's not clear why they didn't.  We're still investigating.  (But we're trying to get a demo going and fixing bugs as we hit them, so it will be a bit before we have the breathing room to delve into the tests.)